ocs_ci/ocs/scale_lib.py

```diff
@@ -1361,6 +1375,7 @@ def attach_multiple_pvc_to_pod_dict(
     io_runtime=None,
     io_size=None,
     pod_yaml=constants.PERF_POD_YAML,
+    remove_security_context_section=False,
```

Comment:
same here, let's keep this value as True by default.

Reply:
yes, np